lib/Installation/ProductSelection/AdditionalProductPage.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Additional Prodcuts page
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::ProductSelection::AdditionalProductPage;
+use parent 'Installation::Navigation::NavigationBase';
```

Comment:
My view it should belong lib/Installation/SystemProbing class since https://openqa.suse.de/tests/9231215#step/dud_addon/1 show background page is system probing.

Reply:
It is only for additional products page. Not related with background page.